pwiz_tools/Skyline/Model/RefinementSettings.cs

```diff
@@ -889,12 +889,12 @@ public enum ConvertToSmallMoleculesChargesMode
             string pathForLibraryFiles, // In case we translate libraries etc
```

Comment:
Is there a benefit to keeping this in RefinementSettings? Or could we move it into its own class in TestUtil?

Reply:
It's test only code, for sure. We put it where it is on purpose, back in the day, but the landscape has grown more complicated. I'll see about moving it, as long as it already has a change.

Reply:
Oops, it's not purely test code - we use portions of it, for example, to display the molecule qualities of a peptide (e.g. chemical formula) in the document grid. So it's not a cheap refactor after all.